scripts/create-dev-env.sh

```diff
@@ -422,7 +422,7 @@ fi
 # compile numpy and scipy if requested
 
 NUMPY_VER="1.6.2"
-SCIPY_VER="0.10.1"
+SCIPY_VER="0.14.0"
```

Comment:
BTW, this script is no longer used as far as we know. I've made a separate pull request to delete it: #13179